src/main/telemetry/hott.h

```diff
@@ -105,18 +106,21 @@ typedef enum {
 #define HOTT_EAM_SENSOR_TEXT_ID  0xE0 // Electric Air Module ID
 #define HOTT_GPS_SENSOR_TEXT_ID  0xA0 // GPS Module ID
 
-
-#define HOTT_TEXTMODE_MSG_TEXT_LEN 168
+#define HOTT_TEXTMODE_DISPLAY_ROWS          8
+#define HOTT_TEXTMODE_DISPLAY_COLUMNS       21
+#define HOTT_TEXTMODE_START                 0x7B
+#define HOTT_TEXTMODE_STOP                  0x7D
+#define HOTT_TEXTMODE_ESC                   0x01
 //Text mode msgs type
-struct HOTT_TEXTMODE_MSG {
-    uint8_t start_byte;  //#01 constant value 0x7b
-    uint8_t fill1;       //#02 constant value 0x00
-    uint8_t warning_beeps;//#03 1=A 2=B ...
-    uint8_t msg_txt[HOTT_TEXTMODE_MSG_TEXT_LEN]; //#04 ASCII text to display to
+typedef struct HOTT_TEXTMODE_MSG_s {
```

Comment:
Should be `hottTextModeMsg_s`, `hottTextModeMsg_t`.

Reply:
Yes! Perhaps someone should rename the other structs in hott.c. ;-)

Reply:
One of the principles I find to be very beneficial to long term code maintainability and quality is 'leave everything at least as good as you found it'. If you modify something that could be better - improve it. If you add something new, do it right. Cleanup can (and probably should be) done in a separate pull request, and I understand if it isn't done, because we all do this for a hobby.